google/cloud/forseti/services/inventory/base/cai_gcp_client.py

```diff
@@ -52,23 +92,101 @@ def session(self):
         self._local.cai_session = db.create_readonly_session(engine=self.engine)
         return self._local.cai_session
 
-    def iter_compute_backendservices(self, project_number):
-        """Iterate Backend services from Cloud Asset data.
+    def fetch_billing_account_iam_policy(self, account_id):
+        """Gets IAM policy of a Billing Account from GCP API.
```

Comment:
`from GCP API` would not be entirely right?  Since it would first attempt to use `self.dao.iter_cai_assets()`, and then the GCP API??  How about `from Cloud Asset data or GCP API`?